src/protocol/testser.c

```diff
@@ -64,9 +64,10 @@ void check_options(u8 init) {
         speed = Model.proto_opts[TESTSER_SPEED];
         switch(speed) {
         case 0: UART_SetDataRate(115200); break;
-        case 1: UART_SetDataRate(9600); break;
-        case 2: UART_SetDataRate(100000); break;
-        case 3: UART_SetDataRate(400000); break;
+        case 1: UART_SetDataRate(57600); break;
```

Comment:
can we keep the original order but add 57600 as last one? It is always good to not change user experience unless we really need.

Reply:
This is only used for developer purposes anyways. We might as welk sort ascending or descending by bps

Reply:
And TESTSER doesn't work in the dev build because dev grabs the serial port for printf.  There's really not much need for TESTSER since SBUS and CRSF are now available.  The only advantage of TESTSER is the loopback mode.